core/profile_manager.py

```
@@ -164,20 +173,64 @@ def migrate_single_config_to_default() -> None:
+                )
+                return
             sbak = base_dir / "secrets.toml.pre-profile.bak"
             old_secrets.rename(sbak)
-        except Exception:
-            pass
+            _log.info(
+                "Migrated legacy secrets %s -> %s",
+                old_secrets,
+                default_sec,
+            )
+        except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)